src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -215,7 +215,17 @@ private static void OnUIElementKeyUp(object sender, KeyRoutedEventArgs e)
 			var target = GetAutoFocusNextElement(host);
 			if (GetAutoFocusNext(host) || target != null) // either property can be used to enable this feature
 			{
-				target ??= FocusManager.FindNextElement(FocusNavigationDirection.Next, new FindNextElementOptions { SearchRoot = host }) as Control;
+				if ((host as UIElement)?.XamlRoot?.Content is { } rootContent)
+					target ??= FocusManager.FindNextElement(FocusNavigationDirection.Next,
+						new FindNextElementOptions
+						{
+							SearchRoot = (host as UIElement)?.XamlRoot?.Content as DependencyObject ?? host
+						}
+					) as Control;
+				else
+					_logger.Warn(
+						$"AutoFocusNext: cannot move focus because XamlRoot.Content is null for host={host.GetType().Name}"
+					);
```

Comment:
add braces for the if/else blocks, just to match convention :)

Reply:
I'm gonna be annoying here too and also request that we use this opportunity to add runtime tests for the inputextensions: https://github.com/unoplatform/uno.toolkit.ui/tree/main/src/Uno.Toolkit.RuntimeTests/Tests :)